tools/omero/omero_filter.py

```diff
@@ -0,0 +1,84 @@
+import argparse
+import csv
+import json
+import sys
+
+import ezomero as ez
+
+
+def filter_ids_ezo(user, pws, host, port, filter, id, value1, value2=None, tsv_file="filter_list.tsv"):
+
+    # Transform the id input in a list of integer
+    id = id.split(',')
+    id = list(map(int, id))
+
+    # Function to write tabular file from the ezomero output
+    def write_ids_to_tsv(data):
+        with open(tsv_file, 'w', newline='') as f:
+            writer = csv.writer(f, delimiter='\t')
+            for item in data:
+                writer.writerow([item])  # Write each ID
+
+try:
+    with ez.connect(user, pws, "", host, port, secure=True) as conn:
+        try:
+            if filter == "filename":
+                fn_ids = ez.filter_by_filename(conn, id, value1)
+                write_ids_to_tsv(fn_ids)
+                return fn_ids
+
+            elif filter == "KP":
+                kp_ims = ez.filter_by_kv(conn, id, value1, value2)
+                write_ids_to_tsv(kp_ims)
+                return kp_ims
+
+            elif filter == "tag":
+                tg_dict = ez.filter_by_tag_value(conn, id, value1)
+                write_ids_to_tsv(tg_dict)
+                return tg_dict
+
+            else:
+                raise ValueError(f"Unsupported object type: {filter}")
+
+        except ValueError as ve:
+            sys.exit(f"ValueError: {str(ve)}")
+
+except Exception as e:
+    sys.exit(f"Connection error: {str(e)}")
```

Comment:
Was thinking that if all we do with the exception is to print it, then we can just skip it. The exception will be printed anyway and wa also will have a non-zero exit code.

```suggestion
```